### PR TITLE
Fix tagging import

### PR DESCRIPTION
**Pull request by CalebBredekamp**

### CODE REVIEW

File: Service/src/main/java/za/co/imqs/coreservice/imports/Rules.java [hunk 1]
```diff
@@ -219,22 +219,26 @@ public void setParameterString(String value) {
             with = s[1];
         }
     }
-
-    // TODO: Can be made more general with a robust way of containing two regular expressions in one string
-    public class RemoveBraces implements StringProcessor {
+    
+    public class PrepareTags implements StringProcessor {
         private String reg;
```
Comment (CalebBredekamp) [inline Service/src/main/java/za/co/imqs/coreservice/imports/Rules.java:224]: Will remove

File: Service/src/main/java/za/co/imqs/coreservice/imports/Rules.java [hunk 2]
```diff
@@ -219,6 +219,33 @@ public void setParameterString(String value) {
...
+            }
+
+            value = value.trim().replaceAll("\\{(.+)\\}", "$1");
+
+//            To pass the loadSmallSet() test the values must be reversed
+//            String[] arr = value.split(",");
+//            ArrayUtils.reverse(arr);
+//            value = String.join(",", arr);
+
+            value = value.replaceFirst(",", "?");
```
Comment (frankvanrietimqs) [inline Service/src/main/java/za/co/imqs/coreservice/imports/Rules.java:239]: It is a metter of personal preference I guess. But there is a logical distinction between removing the curly brackets etc. above and  formatting the data to format it as HTTP parameters. If it were me I would do the latter at the point where the data is sent as the format is dependent on the transport mechanism. It is up to you if you want to change this or not.

**Review by frankvanrietimqs — COMMENTED** [review]
Looks good. Please just also update the documentation that relates to tags and importing.
E.g. https://github.com/IMQS/Core/blob/master/Service/IMPORT.md and https://github.com/IMQS/Core/blob/master/Service/deploy/index.html

File: Service/src/main/java/za/co/imqs/coreservice/imports/Rules.java (hunk 2 above)
Comment (CalebBredekamp) [reply Service/src/main/java/za/co/imqs/coreservice/imports/Rules.java:239]: Yeah, no I agree with that. I'll change it

File: Service/deploy/index.html [hunk 3]
```diff
@@ -13,7 +13,9 @@ <h1>Welcome to the download page</h1>
 You need to download all 4 files for your OS and then run either <tt>import.bat</tt> or <tt>import.sh</tt>.
 </p>
 
-<h2>Commandline</h2>
+<h2>Importing</h2>
```
Comment (frankvanrietimqs) [inline Service/deploy/index.html:16]: Cool makes sense to add that header. Having adde4d this I think that text text preceding this new header should move into this section as it deals with imports.

**Review by frankvanrietimqs — APPROVED** [review]

**Pull request merged**